test/lib.bash

```diff
@@ -40,7 +40,7 @@ function print_test_result {
   fi
 }
 
-function run_e2e_tests {
+function downstream_e2e_tests {
```

Comment:
What about `serverless_operator_e2e_tests` ? Strictly speaking we don't need to say it's downstream because for this there's no upstream. Or maybe just `e2e_tests` .  The rest of changes looks good.

Reply:
`serverless_operator_e2e_tests` sounds better and consistent with other function names.